docs/howto/configure-global-pull-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+tags:
+  - alpha
+---
+
+# Configure global pull secrets for allowing components to pull private images
+
+**Note: The UX for how auth info for using private images is provided is an active work in progress.**  
+
+To configure `catalogd` and `operator-controller` to use authentication information for pulling private images (catalog/bundle images etc), the components can be informed about a kubernetes `Secret` object that contains the relevant auth information. The `Secret` must be of type `kubernetes.io/dockerconfigjson`. 
+
+Once the `Secret` is created, `catalogd` and `operator-controller` needs to be redeployed with an additional field, `--global-pull-secret=<secret-namespace>/<secret-name>` passed to the respective binaries.
+
+For eg, create a `Secret` using locally available `config.json`: 
+
+```sh
+$ kubectl create secret docker-registry test-secret \
+  --from-file=.dockerconfigjson=$HOME/.docker/config.json \
+  --namespace olmv1-system
+secret/test-secret created
```

Comment:
Should we include the output of the command? Adding it might make it harder for users to copy, paste, and use the command directly, right?